src/SqlConverters.cs

```diff
@@ -68,16 +69,20 @@ internal class SqlGenericsConverter<T> : IAsyncConverter<SqlAttribute, IEnumerab
         {
             private readonly IConfiguration _configuration;
 
+            private readonly ILogger _logger;
+
             /// <summary>
             /// Initializes a new instance of the <see cref="SqlGenericsConverter<typeparamref name="T"/>"/> class.
             /// </summary>
             /// <param name="configuration"></param>
+            /// <param name="logger">ILogger used to log information and warnings</param>
             /// <exception cref="ArgumentNullException">
             /// Thrown if the configuration is null
             /// </exception>
-            public SqlGenericsConverter(IConfiguration configuration)
+            public SqlGenericsConverter(IConfiguration configuration, ILogger logger)
             {
                 this._configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+                this._logger = logger;
```

Comment:
null check?

Reply:
I don't think it's needed since we create the logger in the provider and we have a null check for the logger factory there.